mlflow/tensorflow.py

```diff
@@ -694,17 +694,57 @@ def _setup_callbacks(lst):
 def autolog(every_n_iter=100):
     # pylint: disable=E0611
     """
-    Enable automatic logging from TensorFlow to MLflow. If applicable,
-    model checkpoints are logged as artifacts to a 'models' directory, along
-    with any TensorBoard log data.
+    Enables automatic logging from TensorFlow to MLflow.
+    Note that autologging for ``tf.keras`` is handled by :py:func:`mlflow.tensorflow.autolog`,
+    not :py:func:`mlflow.keras.autolog`.
+    As an example, try running the
+    `TensorFlow examples <https://github.com/mlflow/mlflow/tree/master/examples/tensorflow>`_.
 
-    Refer to the tracking documentation for
-    information on what is logged with different TensorFlow workflows.
+    For each TensorFlow module, autologging captures the following information:
+
+    **tf.keras**:
+     - **Metrics** and **Parameters**:
+
+      - Training loss; validation loss; user-specified metrics
+      - ``fit()`` or ``fit_generator()`` parameters; optimizer name; learning rate; epsilon
+
+     - **Artifacts**:
+
+      - Model summary on training start
+      - `MLflow Model <https://mlflow.org/docs/latest/models.html>`_ (Keras model)
+      - TensorBoard logs on training end
+
+    **tf.keras.callbacks.EarlyStopping**:
+     - **Metrics** and **Parameters**:
+
+      - Metrics from the ``EarlyStopping`` callbacks: ``stopped_epoch``, ``restored_epoch``,
+        ``restore_best_weight``, etc
+      - ``fit()`` or ``fit_generator()`` parameters associated with ``EarlyStopping``:
+        ``min_delta``, ``patience``, ``baseline``, ``restore_best_weights``, etc
+
+    **tf.estimator**:
+     - **Metrics** and **Parameters**:
+
+      - TensorBoard metrics: ``average_loss``, ``loss``, etc
+      - Parameters ``steps`` and ``max_steps``
+
+     - **Artifacts**:
+
+      - `MLflow Model <https://mlflow.org/docs/latest/models.html>`_ (TF saved model) on call
+        to ``tf.estimator.export_saved_model``
+
+    **TensorFlow Core**:
+     - **Metrics**:
+
+      - All ``tf.summary.scalar`` calls
+
+    Refer to the autologging tracking documentation for
+    information on what is the behavior of `TensorFlow workflows
+    <https://www.mlflow.org/docs/latest/tracking.html#tensorflow-and-keras-experimental>`_.
 
     :param every_n_iter: The frequency with which metrics should be logged.
```

Comment:
In the screenshots, it looks like the `Parameters` subheader has the same style as the subheaders for the different modules listed earlier (e.g. `TensorFlow Core`). Is there any way to meaningfully differentiate the headers to make it clear we're done with listing modules?

Reply:
Note two things:

- the sentence following TF 1.x and TF2.x refers to link to the TF examples
- The last sentence after `TensoreCore:` module signals that this is the end of the modules and frameworks. In way, it dilneates  this from the `Parameters` list as part of the argument to the method.

Reply:
Ah ok I see what you mean. In that case I think we should be ok leaving it as is. Thanks for looking into it!